docs/metrics.md

```diff
@@ -191,6 +222,12 @@ Returns the amount of space in bytes restored from the source virtual machine. T
 ### kubevirt_vmsnapshot_disks_restored_from_source_total
 Returns the total number of virtual machine disks restored from the source virtual machine. Type: Gauge.
 
+### kubevirt_vmsnapshot_persistentvolumeclaim_labels
+Returns the labels of the persistent volume claims that are used for restoring virtual machines. Type: Info.
+
+### kubvirt_number_of_vms
```

Comment:
@machadovilaca I think it should be:
The number of VMs in the cluster by namespace. Type: Gauge.

Reply:
updated